### ISSUE TITLE
How to reduce model size that chosen yolov8 models?

### ISSUE THREAD

**Issue by madenburak** [ultralytics/ultralytics#782]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

I have trained yolov8n model for simple case with two classes. But metric scores achieving up to %90 before 10th epoch. And train loss is flatting. Images look likes each other in dataset. There is imbalance between classes.

- I deacreased learning rate to 0.2*lr and 0.2*lrf.
- I reduce model nodes but i didn't touch to depth and width

* Now, I will crop  bbox  from class that icluding little data images then add to dataset. 
* I want to use dropout but I didn't find for detection, there is for classify only.

What is your suggesstion for me?

### Additional

![image](https://user-images.githubusercontent.com/33545676/216443265-d2f56556-88af-4a67-a1d3-9667660a80ff.png)

**Comment by AyushExel** [comment]
I didn't quite get the question exactly. But I'll try to answer with what I've understood. You want to reduce the model size of the existing models. You can do that by editing the official yaml files if you're able to find your way around the codebase. It is not a well-documented workflow. We plan to do this later this year.
